package.json

```diff
@@ -21,6 +21,9 @@
   "dependencies": {
     "builder": "~2.4.0",
     "builder-radium-component": "^0.2.2",
```

Comment:
There's an ongoing discussion around `dependencies` vs. `devDependencies` for frontend libraries. To cut down on installation time, I am pushing for including all build-time packages in `devDependencies`, and only put whatever is needed for `lib/` usage in real `dependencies. Wherever we build the `libs` and `dist` files, we'd do something like this:

```
NODE_ENV=development npm install
NODE_ENV=production builder concurrent build-dist build-lib
```

 `builder` and `builder-radium-component` should therefore go in `devDependencies`.